packages/grid/examples/preview/src/components/App.js

```diff
@@ -158,6 +158,37 @@ const sections = [
       },
     ],
   },
+  {
+    name: 'Hide column per breakpoint',
+    description:
+      'Use the column helpers to specify a column span of 0 at different widths',
+    content: () => (
```

Comment:
I _think_ this isn't mirroring the `{ name, description, examples }` structure used by the other examples which is causing the preview for this example to error out. Mind updating it real quick to match the other structures so we can render the preview? 👀

Reply:
I think I messed up and copied it into the wrong location. I meant to have it as part of the "Usage" section. The similar structure of nested objects probably tripped me up. Fixed it here: 267b28f